samples/browser-script-vite-vue/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<link rel="icon" href="/favicon.ico" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>Vite App</title>
+		<link
+			rel="stylesheet"
+			data-name="vs/editor/editor.main"
+			href="https://unpkg.com/monaco-editor@0.32.1/min/vs/editor/editor.main.css"
+		/>
+		<script src="https://unpkg.com/monaco-editor@0.32.1/min/vs/loader.js"></script>
+		<script src="https://unpkg.com/monaco-editor@0.32.1/min/vs/editor/editor.main.nls.js"></script>
+		<script src="https://unpkg.com/monaco-editor@0.32.1/min/vs/editor/editor.main.js"></script>
```

Comment:
Hmm, I'm not sure if this is a good way to load the monaco editor. Ideally the monaco editor is distributed locally and not loaded from a CDN. Examples should not encourage to do that.